lib/broadway/topology.ex

```diff
@@ -11,6 +11,12 @@ defmodule Broadway.Topology do
     RateLimiter
   }
 
+  defmodule Config do
+    @moduledoc false
+
+    defstruct [:context, :topology, :producer_names, :batchers_names, :rate_limiter_name]
+  end
```

Comment:
Instead of defining a separate module, we can use the topology module itself:

```suggestion
  defstruct [:context, :topology, :producer_names, :batchers_names, :rate_limiter_name]
```

Less work for the compilers. ;)